arches_querysets/datatypes/resource_types.py

```diff
@@ -0,0 +1,181 @@
+import json
+import logging
+import uuid
+
+from arches import VERSION as arches_version
+from arches.app.datatypes import datatypes
+from arches.app.models import models
+
+from django.utils.translation import get_language, gettext as _
+
+
+logger = logging.getLogger(__name__)
+
+
+class ResourceInstanceDataType(datatypes.ResourceInstanceDataType):
+    def transform_value_for_tile(self, value, **kwargs):
+        def from_id_string(uuid_string, graph_id=None):
```

Comment:
naming nit here. I get having something named like this on a class eg `ResourceInstanceDataType.from_id_string()` but as a private function it reads oddly for me.